integration/deterministic-build-output-test.ts

```diff
@@ -60,7 +60,7 @@ test("builds deterministically under different paths", async () => {
         },
       }),
 
-      "postcss.config.js": js`
+      "postcss.config.mjs": js`
```

Comment:
Something puzzling is happening in this seeming unrelated test.
After installing `@vanilla-extract/vite-plugin` in `integration/package.json`, this test fails (not only on CI but it also fails locally on my PC).

- https://github.com/remix-run/remix/actions/runs/6861294763/job/18656701531#step:7:424

```
✘ [ERROR] require() of ES Module /home/runner/work/remix/remix/.tmp/integration/remix-node-template-s04nobi5qlo/postcss.config.js from /home/runner/work/remix/remix/node_modules/lilconfig/dist/index.js not supported.
```

My hunch is that `@vanilla-extract/vite-plugin` has dependency on `postcss-load-config` and the existence of this in the visible `node_modules` might be affecting some postcss's internal config resolution, but `postcss-load-config` v3 seems to have issue with esm https://github.com/postcss/postcss-cli/issues/387#issuecomment-1150819626.

Reply:
It seems moving `postcss-load-config` to root fixes it somehow https://github.com/remix-run/remix/pull/8000/commits/4222ed94392a4de5d03d234e3157816e155c234b though I don't feel I'm doing the right thing...
Please let me know if it should be handled in a different way.

Reply:
Great find. I opened a PR to fix this at the source https://github.com/vanilla-extract-css/vanilla-extract/pull/1231

Reply:
Thanks !
I don't't have much experience with postcss, so I was quite at lost.
I would be happy to test if the fix is released. Please feel free to let me know!